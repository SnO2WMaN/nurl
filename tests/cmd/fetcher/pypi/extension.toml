args = ["https://pypi.org/project/nltk", "3.8", "--arg-str", "extension", "zip"]
