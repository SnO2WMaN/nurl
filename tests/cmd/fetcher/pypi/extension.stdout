fetchPypi {
  pname = "nltk";
  version = "3.8";
  hash = "sha256-dLMIJqN9eNU0JxBbvQN92IAlG+Jp/KZO5TCDikbtVfw=";
  extension = "zip";
}